### PR TITLE
Added debugging configuration

### PR DESCRIPTION
**Pull request by ernesizcl88**

### CODE REVIEW

File: test/fragment-generation-utils-test.js [hunk 1]
```diff
@@ -831,6 +839,10 @@ describe('FragmentGenerationUtils', function() {
   });
 
   it('will halt generation after a certain time period', function() {
+    if(isDebug()) {
```
Comment (tfmar) [inline test/fragment-generation-utils-test.js:842]: Style guide requires a space: `if (isDebug()) {`

https://google.github.io/styleguide/jsguide.html#formatting-horizontal-whitespace

I think running `npm install` will automatically fix these things for you.

**Review by tfmar — APPROVED** [review]
lgtm (pending fix for formatting)

File: test/fragment-generation-utils-test.js (hunk 1 above)
Comment (tomayac) [reply test/fragment-generation-utils-test.js:842]: `npm run fix` does the trick.

Comment (ernesizcl88) [reply test/fragment-generation-utils-test.js:842]: thank you both!

**Pull request merged**